lmn/api_calls.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from .models import Artist, Venue
+
+
+def search_mb_place(place):
+    """ searches musicbrainz API for a venue and returns a list of matches some results will not have an address
+    so there is a try/except to handle for now """
+    url = f'http://musicbrainz.org/ws/2/place/?query=place:{place}'
+    headers = {"User-Agent": "LMNApp/1.0(sean@dropanddram.com)", "Accept": "application/json"}
+    params = {
+        'fmt=': 'json',
+        'limit': 10
+    }
+    data = requests.get(url, headers=headers, params=params).json()
+    places = data['places']
+    result_list = []
+    for place in places:
+        new_name = place['name']
+        try:
+            new_address = place['address']
+        except KeyError as e:
```

Comment:
You can use .get('address') which will return None if key not found, instead of trying to get the key and catching a key error. Or use get with a default value used if the key is not present https://docs.python.org/3/library/stdtypes.html?highlight=get%20dict#dict.get